app/workers/articles/dev_rss_reader_worker.rb

```diff
@@ -0,0 +1,17 @@
+# NOTE: [rhymes]
+# This script will soon be removed. We need it to collect monitoring data on
+# Datadog about the production behavior of the `RssReader` class
+module Articles
+  class DevRssReaderWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority, retry: 10
+
+    def perform
+      return unless SiteConfig.community_name == "DEV"
+
+      # we force fetch to have realistic data
+      RssReader.get_all_articles(force: true)
```

Comment:
Since we dont want to end up in an infinite retry loop if for some reason this keeps restarting(Once Sidekiq jobs are running in our system there is no way to cancel them) due to deploys, let's add a little "quick exit" clause here that we could trigger if we want the worker to stop being enqueued. 
```ruby
return if Rails.cache.read("cancel_rss_job").present?
```

Reply:
Love the escape hatch! I'll add it